packages/clerk-js/package.json

```diff
@@ -47,6 +47,9 @@
     "@emotion/cache": "11.10.5",
     "@emotion/react": "11.10.5",
     "@floating-ui/react": "0.19.0",
+    "@zxcvbn-ts/core": "^2.2.1",
+    "@zxcvbn-ts/language-common": "^2.0.1",
```

Comment:
The implementation is using those, why do they need to be removed ?

Reply:
Don't we lazy load them?

Reply:
Yes we do, but based on this implementation, we need to bundle the dependencies ourselves, so this is needed right ?

If we end up loading them with `loadScript` we can remove these from package.json

Reply:
My comments came from the fact that I was thinking that either we will use loadScript or `import(/*webpackIgnore: true*/ 'cdn.jsdelivr.net/path/to/module.js');` so that the package doesn't passes from our services or webpack.